### PR TITLE
DSO-613 Installer support for the OIDC microservice

### PR DESCRIPTION
**Pull request by david-oc-miller**

### CODE REVIEW

File: vagrant/provisioning/roles/arkcase-app/tasks/main.yml [hunk 1]
```diff
@@ -911,6 +911,15 @@
     dest: /home/arkcase/.arkcase/acm/acm-config-server-repo/arkcase-commons-server.yaml
   when: arkcase_version == "" or arkcase_version is version('2021.04', '>=')
 
+- name: server-specific configuration for arkcase-portal-oidc.yml for version 2021.04
```
Comment (martinbozinovski) [inline vagrant/provisioning/roles/arkcase-app/tasks/main.yml:914]: As i am aware of, ArkCase is at 2022.01-RC1. Are we going with 2021.04?

**Review by martinbozinovski — COMMENTED** [review]
Other that arkcase_version, being 2021.04, this looks good!.

File: vagrant/provisioning/roles/arkcase-app/tasks/main.yml (hunk 1 above)
Comment (david-oc-miller) [reply vagrant/provisioning/roles/arkcase-app/tasks/main.yml:914]: 2021.04 is the convention used in other version checks for any version higher than 2021.03, including 2022 and future versions.

Comment (martinbozinovski) [reply vagrant/provisioning/roles/arkcase-app/tasks/main.yml:914]: Perfect than. Thanks for the clarification!.

**Review by martinbozinovski — APPROVED** [review]

**Review by mariogjurcheski — APPROVED** [review]

**Pull request merged**